src/control/server/mgmt.go

```diff
@@ -78,11 +77,57 @@ func (c *controlService) Teardown() {
 		log.Debugf(
 			"%s\n", errors.Wrap(err, "Warning, SCM Teardown"))
 	}
+}
 
-	// decrement counter to release blocked goroutines
-	for idx := range c.config.Servers {
-		c.config.Servers[idx].storWaitGroup.Done()
+// awaitStorageFormat checks if running as root and server superblocks exist,
```

Comment:
Do we want to handle a timeout here so that this can return with an error if the format doesn't occur in time?  Or is there no architectural need at this time?

Reply:
Comment applies to the function call awaitStorageFormat.

Reply:
no I think the idea is for an indefinite wait for administrator intervention